### PR TITLE
Include workspace click events for /workspaces

### PR DESCRIPTION
**Pull request by jakobhero**
Include events for the click that caused a workspace start from the dashboard in order to understand user behaviour better.

/werft analytics=segment|TEZnsG4QbLSxLfHfNieLYGF4cDwyFWoe

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/4913#" title="Author self-approved">jakobhero</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **corneliusludmann** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @corneliusludmann` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jakobhero** [comment]
cc @svenefftinge

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx
Comment (csweichel) [inline components/dashboard/src/workspaces/WorkspaceEntry.tsx:49]: this is inconsistent with the rest of the system. We've been quite consistent in writing `workspaceId` when it comes to logging and tracing. Also most analytics events already produce `workspaceId`. I'd highly recommend we follow the same naming convention here.

**Review by svenefftinge — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 2fa5c8774bd23238472cf8fb4df9463eb52de307</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/4913#pullrequestreview-722116149" title="Approved">svenefftinge</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [svenefftinge]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jakobhero** [comment]
Closing this as objective is already fulfilled in #5168

**Pull request closed without merging**
